.github/workflows/build-executable.yaml

```
@@ -86,72 +93,95 @@
+        env:
+          REPO_B_OWNER: PaladinCloud  # Replace with Repo B owner
+          REPO_B_NAME: installer # Replace with Repo B name
+          WORKFLOW_FILE_NAME: deploy-dev.yaml  # Replace with Repo B workflow filename
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Ensure this token has access to Repo B
         run: |
-          aws s3 cp --recursive asset-sender-1.0-SNAPSHOT.jar s3://${{ secrets.AWS_BUCKET_QA }}/${{ needs.extract_version.outputs.version }}/qa/deltaengine
+          curl -X POST \
+            -H "Authorization: Bearer $GITHUB_TOKEN" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/$REPO_B_OWNER/$REPO_B_NAME/actions/workflows/$WORKFLOW_FILE_NAME/dispatches \
+            -d '{"ref": "main}'  # Adjust branch if necessary
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)